controllers/phish_test.go

```diff
@@ -63,6 +71,19 @@ func (s *ControllersSuite) TestOpenedPhishingEmail() {
 	s.Equal(result.Status, models.EVENT_OPENED)
 }
 
+func (s *ControllersSuite) ReportedPhishingEmail() {
+	campaign := s.getFirstCampaign()
+	result := campaign.Results[0]
+	s.Equal(result.Status, models.STATUS_SENDING)
+
+	s.reportedEmail(result.RId)
+
+	campaign = s.getFirstCampaign()
+	result = campaign.Results[0]
+	s.Equal(result.Status, models.EVENT_OPENED)
+//	s.Equal(result.Status, models.EVENT_REPORTED)
```

Comment:
Should this be commented out? It looks like we would expect the `EVENT_REPORTED` event to have fired here.

Reply:
For this one, no. I thought I'd fully removed that, must have missed it. I had originally had 'reported' as a status, so this was a remnant from that. I separated it out as a distinct thing though, hence removing that line. I left reporting an email also setting it as being opened as:

1. Opening isn't a bad thing
2. It gives a more thorough idea of the amount of emails which get delivered and seen by users. If the tracking image is blocked you could end up with more reports than opened, which seemed odd to me.